modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeVelocityIfStatementsMigrationCheck.java

```diff
@@ -154,12 +105,6 @@ private boolean _hasBreakLine(String line) {
 
 	private static final String _VELOCITY_ELSEIF_START = "#elseif";
 
-	private static final String _VELOCITY_END = "#end";
-
-	private static final String _VELOCITY_FOREACH_START = "#foreach";
-
 	private static final String _VELOCITY_IF_START = "#if";
```

Comment:
@nicolas-sss, this same variable is created in the class `VelocityMigrationUtil`, in light of this how about creating a new Java class to hold all constants reused across the checks?